### ISSUE TITLE
Deep-All results on PACS

### ISSUE THREAD

**Issue by wangtingwei1993** [wildphoton/RandConv#1]
Hi, Thanks for the interesting work. 

You said in your paper that your Deep-All results on PACS are not directly comparable with Jigen due to different implementations. Could you please clarify what are the differences between your implementation and JiGen? You know your Deep-All result is almost 5% (on Avg.) lower than that of Carlucci's implementations.  In addition, do you know what causes this difference? augmentation, model initialization, optimizer, learning rate decay strategy, or something else?

Thanks a lot.

**Comment by wildphoton** [comment]
@wangtingwei1993 Thanks for your interest! I think the major source difference is that we are using the official data splits for train/val/test while JiGen made their own splits. JiGen also uses random cropping/flip augmentation while ours does not. I think we are all using the PyTorch pretrain weights. Optimizer, LR decay and batch size are probably not the exactly same. But I won't expect them to make a huge difference. I believe the data splits (maybe + extra augmentations) made the JiGen results not the same. I will update our PACS scripts soon.
